readme.md

```diff
@@ -28,7 +28,8 @@ npm install neo-proxy --save
     port: 8888,
     target: 'http://dtrejo.com',
     needle: '</body>',
-    customHTML: '<script>alert("Hello, it's working! — DTrejo")</script>'
+    customHTML: '<script>alert("Hello, it's working! — DTrejo")</script>',
+    rewriteUrlsHost: 'localhost' // or '10.1.1.250'
```

Comment:
- Could you write a sentence or two about the purpose of rewriteUrlsHost? Why someone would want to use it, how to disable it, etc.
- Maybe change the name of the option so it is more obvious?

While you're in here, would you remove the `it's` contraction? The quote makes it invalid JS, my fault.